ml4h/tensormap/ukb/mri.py

```diff
@@ -5,8 +5,8 @@
 import h5py
 import numpy as np
 from tensorflow.keras.utils import to_categorical
-import cv2  # TODO: Add to ML4H docker image
-import blosc  # TODO: Add to ML4H docker image
+import cv2
+import blosc
```

Comment:
Note that I only cleaned up these two comments. All other changes to this file were made by the pre-commit.